panacea/web/static/js/view.js

```diff
@@ -263,7 +263,7 @@ export const displayError = (error, pml) => {
   errorsBadge.innerHTML = '1';
   errorsTab.click();
   errorTitleElement.innerHTML = error.title;
-  errorTextElement.innerHTML = error.detail;
+  errorTextElement.innerHTML = `<strong>${error.detail}</strong>`;
```

Comment:
Some of the call sites already wrap the `detail` in a strong or h5. Should probably remove those.